openassessment/templates/openassessmentblock/oa_submission_answer.html

```diff
@@ -8,7 +8,11 @@ <h5 class="submission__answer__part__text__title">{% trans "The question for thi
             </div>
             <article class="submission__answer__part__prompt">
                 <div class="submission__answer__part__prompt__value">
-                    {{ part.prompt.description|linebreaks }}
+                    {% if prompts_type == 'html' %}
+                        {% autoescape off %}{{ part.prompt.description }}{% endautoescape %}
```

Comment:
This might be cleaner with:
```{{ part.prompt.description|safe }}```
You don't need the autoesape off and on

Reply:
yes. you are right. I've fixed this